pallets/gear/src/lib.rs

```diff
@@ -188,8 +188,15 @@ pub mod pallet {
         /// Balances management trait for gas/value migrations.
         type Currency: LockableCurrency<Self::AccountId> + ReservableCurrency<Self::AccountId>;
 
+        /// Multiplier used for conversion from gas into balance.
+        #[pallet::constant]
+        type GasToBalanceMultiplier: Get<BalanceOf<Self>>;
```

Comment:
This has got exposed, but in fact I do not need it for the implemented functionality

Reply:
As it become useless I'd remove the stuff cause it may cause some compatibility problems in future for JS side, metadata etc, as less thing we throw in config as better it is

Reply:
See no reason to keep it now